versionutils/version.go

```diff
@@ -21,6 +21,7 @@ const (
 // SpeciallyOrderedPrefixes are prefixes that are ordered in a special way
 // This is exported so that consuming packages can specify their own special labels
 // These formats take precedence over alphanumeric ordering for labels
+// Labels take are ordered in increasing order of precedence
```

Comment:
Isn't this... decreasing order of precedence? 
These take precedence over other labels, and "rc" takes precedence over "dev".